modules/users/data/data.yaml

```diff
@@ -1,10 +1,10 @@
 groups:
-  ops:
+  infrastructure-engineers:
```

Comment:
renaming groups is a manual process, personally I don't see why it should be renamed.

Also root on all server == site reliability engineers, they're not 'infrastructure engineers'

Reply:
Your proposal splits us into 2 groups. My understanding is those with root everywhere would be infrastructure engineers.

Reply:
Split into two departments, Site Reliability Engineers is an appropriate job description of what the Infrastructure team are, it’s about keeping role descriptors relevant which is why MediaWiki Administrators are proposed to not be called ‘Administrators’ because that’s not a good descriptor. They also can’t really be called Site Reliability Engineers because they don’t have the wider toolkit to manage the core infrastructure behind the MediaWiki stack. They also can’t be called Software Engineers because most aren’t active MediaWiki code developers and don’t have a primary focus on code development over sysadmin work

Reply:
ops != sre but it's a point of discussion. If we wish to keep it stating 'ops' or change to 'sre' then that's not an issue, I've deliberately split the changes into multiple commits so individual components can be reverted. It's also a concept to show how it **might** look.

Reply:
For what it's worth, if this is about the commands for MirahezeBots to yelp for help, we should have a more generic command like `!ops` to refer to channel operators, inclusive of, generally but not exclusively, SRE, MediaWiki administrators, stewards, and Global ChanOps (to the extent they continue to be authorized by Group Contacts)

Reply:
This has nothing to do with that. The ops ping stuff for Miraheze bots will come soon but is blocked on me implementing sane memory-cached dynamic configuration dicts.